Sources/BoolExtensions.swift

```diff
@@ -8,10 +8,11 @@
 
 extension Bool {
     /// EZSE: Converts Bool to Int.
-    public var toInt: Int { return Int(self) }
+    public var toInt: Int { return self ? 1 : 0 }
 
     /// EZSE: Toggle boolean value.
-    public mutating func toggle() -> Bool {
+    @discardableResult public mutating func toggle() -> Bool {
```

Comment:
@piv199 thanks for your help! I am trying to answer question by my poor english. 😂